portfolio/src/main/java/com/google/sps/servlets/ExperienceServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/** Servlet that responds with the current date. */
+@WebServlet("/experience")
```

Comment:
Optional: Rename, this doesn't really match the servlet behavior.

Note, updating servlet names means you'll need to update any JS calls to reference the new URL

Reply:
Renamed 👍